test_rails/generators/flipper/setup_generator_test.rb

```diff
@@ -0,0 +1,64 @@
+require "helper"
+require "generators/flipper/setup_generator"
+
+class SetupGeneratorTest < Rails::Generators::TestCase
+  tests Flipper::Generators::SetupGenerator
+  ROOT = File.expand_path("../../../tmp/generators", __dir__)
+  destination ROOT
+  setup :prepare_destination
+
+  test "invokes flipper:active_record generator if ActiveRecord adapter defined" do
+    begin
+      load 'flipper/adapters/active_record.rb'
+      run_generator
+      assert_migration "db/migrate/create_flipper_tables.rb"
+    ensure
+      Flipper::Adapters.send(:remove_const, :ActiveRecord)
+    end
+  end
+
+  test "does not invoke flipper:active_record generator if ActiveRecord adapter not defined" do
+    # Ensure adapter not defined
+    Flipper::Adapters.send(:remove_const, :ActiveRecord) rescue nil
+
+    run_generator
+    assert_no_migration "db/migrate/create_flipper_tables.rb"
+  end
+
+  %w(.env.development .env.local .env).each do |file|
+    test "configures Flipper Cloud token in #{file} if it exists" do
+      File.write("#{ROOT}/#{file}", "")
+      run_generator ["--token", "abc123"]
```

Comment:
Really minor, but using `%w(--token abc123)` makes it so that these arguments and options read more like they would on the command line and can more easily be copy/pasted between the two for manual investigation. The `generator` version needs the args and kwargs separated, but with these, it's just nice to have them mirror how it works on the command line.

Reply:
Good call!